adobe/react-spectrum
# fix: remove autoCapitalize to prevent mismatch warning

Closes https://github.com/adobe/react-spectrum/issues/5599

This bug occurs because `autoCapitalize: 'off'` is set for each segment of DateInput, but Firefox defaults to `none`. This mismatch between server-side and client-side rendering causes the warning to be displayed; Safari and Chrome leave it at `off`, so no warning is displayed. DateInput has `input-mode=numeric` and already displays a numeric keyboard, so setting `autoCapitalize` is not necessary.

[A similar error was also occurring](https://github.com/nextui-org/nextui/pull/3376) in NextUI, which is based on react-aria.


## ✅ Pull Request Checklist:


## 📝 Test Instructions:



Open the following URL in Firefox:
https://stackblitz.com/~/github.com/ryo-manba/auto-capitalize-error

1. Click on the link "Page with autoCapitalize mismatch error".
2. Observe the warning in the console.
3. Go back to the home page.
4. Click on the link "Page without autoCapitalize mismatch error" (autoCapitalize is deleted).
5. Observe that no warning is displayed in the console.

## 🧢 Your Project:
